py/dbt-etl.cfg

```diff
@@ -10,6 +10,7 @@
 mysql.exe = /usr/bin/mysql
 node.exe = /usr/bin/node
 publisher.js = /home/ec2-user/BiblePublisher/publish/Publisher.js
+sofria_client.js = /home/vichugof/public_html/fullstacklabs/FCBH/sofria-cli/sofria_mediaId.js
```

Comment:
change the path to /home/ec2-user/sofria-cli/sofria_mediaId.js

Reply:
It has been updated.